stories/2023-02-01-Overwatch-League/OWL_story.ipynb

```diff
@@ -0,0 +1,3730 @@
+{
```

Comment:
<p>Comparing these results with the Team Map Stats shown on the <a href="https://overwatchleague.com/en-us/statslab?statslab=matches" target="_blank">OWL site</a>, it's reassuring to see that they match up exactly.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Thanks for checking!</p>